bazelrio/scripts/deploy/Deploy.java

```diff
@@ -71,6 +76,10 @@ public static void main(String[] args) throws IOException {
         ArrayList<String> dynamicLibraryPaths = parsedArgs.get("dynamic_libraries");
         boolean verbose = parsedArgs.get("verbose");
 
+        if (parsedArgs.get("fast_unsafe_deploy")) {
+            dynamicLibraryPaths = new ArrayList<>();
```

Comment:
I would print out an INFO here. If we get sent a stack trace of deploy failing, having that info would be valuable.